skglm/estimators.py

```diff
@@ -124,7 +124,8 @@ def _glm_fit(X, y, model, datafit, penalty, solver):
             w = np.zeros(n_features + fit_intercept, dtype=X_.dtype)
             Xw = np.zeros(n_samples, dtype=X_.dtype)
         else:  # multitask
-            w = np.zeros((n_features + fit_intercept, y.shape[1]), dtype=X_.dtype)
+            w = np.zeros((n_features + fit_intercept,
```

Comment:
we try to avoid such cosmetic changes : they make the PR diff longer, and so the PR is more complex to review.

Here it's probably your linter which cut the line to make it shorter than 80 characters, a PEP8 requirement.
However in skglm we use lines of max length 88 characters, so this line was fine.
Can you please revert to previous state (see also other occurrences below)?